packages/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -1,11 +1,20 @@
-import type { TabState } from './Tab.types';
+import type { TabSlots, TabState } from './Tab.types';
 
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import { createCustomFocusIndicatorStyle } from '@fluentui/react-tabster';
 import { tokens } from '@fluentui/react-theme';
 import { tabPendingDesignTokens } from '../../tab.constants';
+import { SlotClassNames } from '@fluentui/react-utilities';
 
+/**
+ * @deprecated
+ */
 export const tabClassName = 'fui-Tab';
```

Comment:
For `Tab` you can actually remove this class name instead of deprecating it since the component is still under `unstable`:

```suggestion
```